src/shared/components/ui/Form/ReactSelect/ReactSelect.test.js

```diff
@@ -1,44 +1,43 @@
 import React from 'react'
-import { mount } from 'enzyme'
-import ReactSelect from '@components/ui/Form/ReactSelect'
-import registerIcons from '@helpers/icons'
+import { render, screen } from '@jestutils'
+import ReactSelect from './ReactSelect'
 
-registerIcons()
-
-const props = {
-  id: 'jestem-id',
-  children: 'jestem dziecko',
-  name: 'field-name',
-  value: 'mam-wartosc',
-  field: {
-    name: 'field-name'
-  },
-  form: {
-    errors: {},
-    touched: {}
-  },
-  options: [
-    {
-      value: 'field-name',
-      label: 'test'
-    }
-  ]
-}
+describe('ReactSelect tests', () => {
+  const props = {
+    name: 'select-name',
+    id: 'select-id',
+    options: [{ label: 'label-0', value: 'value-0' }],
+    value: { 'select-name': 'test-value' }
+  }
+  it('properly renders with props', () => {
+    render(<ReactSelect {...props} />)
+  })
+  it('has default func props', () => {
```

Comment:
brakuje odstępu

Reply:
ciekawe czy to można dodać do eslint z możliwością autofixa